org.eclipse.lemminx/src/main/resources/org/eclipse/lemminx/services/snippets/pi-snippets.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think this file should not belong to this PR

Reply:
thanks, removed